third_party/terraform/utils/error_retry_predicates.go

```diff
@@ -67,18 +67,24 @@ func isIoEOFError(err error) (bool, string) {
 	return false, ""
 }
 
-const connectionResetByPeerErr = "connection reset by peer"
-
+const connectionResetByPeerErr = ": connection reset by peer"
```

Comment:
The underlying network error is always specifically something like `$opType $ip -> $ip: $optype: $error` so I think it's ok to try using this error